noisyatom/blogs/models.py

```diff
@@ -0,0 +1,90 @@
+
+from django.conf import settings
+from django.contrib.contenttypes.models import ContentType
+from django.core.urlresolvers import reverse
+from django.db import models
+from django.db.models.signals import pre_save
+from django.utils import timezone
+from django.utils.safestring import mark_safe
+from django.utils.text import slugify
+
+from markdown_deux import markdown
+
+
+'''
+https://docs.djangoproject.com/en/1.11/ref/models/fields/
+'''
+
+class PostManager(models.Manager):
+
+	def active(self, *args, **kwargs):
+		return super(PostManager, self).filter(
+										draft=False).filter(
+													publish__lte=timezone.now())
+
+
+def upload_image_location(instcance, filename):
```

Comment:
What is the purpose of this function? It seems it only creates a string! That's not good quality. Please remove this complexity and create the string in place if it's just 1 line.

Reply:
I have added some comments to read